### PR TITLE
Exempted Holidays

### LINKED ISSUE #154: fine-tune calendar dates
first day: 9/2/2014
last day: 12/8/2014

holidays:
- 11/3/2014
- 11/27/2014
- 11/28/2014

source: http://registrar.columbia.edu/academic-calendar/30

### PR DESCRIPTION
**Pull request by G-ram**
Closes #154

### CODE REVIEW

File: src/js/models/Schedule.coffee [hunk 1]
```diff
@@ -257,6 +253,13 @@ angular.module('Courses.models')
           if subsectionEvent.byday
             courseRRule += ";BYDAY="+subsectionEvent.byday
 
+          courseEXDate = ""
+          for rawDateString in Holidays
+            holidayDate = @formatDateString(rawDateString)
+            courseEXDate += ICSFormatDate(new Date(holidayDate[2],holidayDate[2],holidayDate[2]))+","
```
Comment (natebrennand) [inline src/js/models/Schedule.coffee]: should that be indexes [2, 0, 1], not [2,2,2] ?

**Comment by G-ram** [comment]
Yes.

**Comment by natebrennand** [comment]
this doesn't work for me with gcal when I tested it on gcal.

screenshot 2014-04-15 13 42 25

It's also buggy with the start dates. The first Wednesday class is skipped.
I'm not sure what the best way of resolving this is.

screenshot 2014-04-15 13 39 49

Generated calendar for reference: https://gist.github.com/natebrennand/661c63852ff30f2315f6

**Comment by natebrennand** [comment]
if anyone else wants to test this, make sure to run `bower update` so  the `ics.js` library is updated

**Comment by natebrennand** [comment]
I think the start date needs to be resolved in the library but checking the repeating days of the week vs. the start date.

**Comment by G-ram** [comment]
Should now start on the right day with holidays exempted.

**Comment by natebrennand** [comment]
works for me, can you squash the commits?

**Pull request merged**